language/analysis/src/main/java/com/google/cloud/language/samples/AnalyzeBeta.java

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
We're following the convention of not having  "All Rights Reserved"  license in java-docs-samples